envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadata.kt

```diff
@@ -54,6 +68,13 @@ private fun getCommunicationMode(proto: Value?): CommunicationMode {
     }
 }
 
+fun Value?.toStatusCodeFilter(
+    accessLogFilterFactory: AccessLogFilterFactory
+): AccessLogFilterSettings.StatusCodeFilterSettings? {
+    val value = if (this?.stringValue != null) this.stringValue.toUpperCase() else return null
```

Comment:
Wow, I didn't know you can "else return null".
This returns from function, not from if, right? Please simplify the control flow according to Kotlin idiomatic style. I'm sure this is easier if you use `?` properly.

Reply:
fixed